### PR TITLE
chore: Restructure charts v8 package.

### PR DESCRIPTION
**Pull request by AtishayMsft**
## Previous Behavior
Moving react-charting package inside a charts outer structure.
The goal is to accommodate an incoming v9 version of the charts control. (PR #32966)

The new structure will look like this.
/-charts
/---/ react-charting (v8)
/---/ react-charts (v9) 

With this restructuring we would be able to build v8, v9 and web component versions of our chart control simultaneously.
We plan to add a charts core package that will contain pure geometrical logic without the UI layer that will be used across different platforms and versions of the charting library.

## Related Issue(s)

- Fixes #

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -365,6 +365,7 @@
       "packages/fluentui/*",
```
Comment (fabricteam) [inline package.json:365]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33145&commitId=43060b179863d61e4857731f22dfe4e58620068c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|284| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

Comment (fabricteam) [inline package.json:365]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33145&commitId=43060b179863d61e4857731f22dfe4e58620068c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom auto edge.chromium.png|2309| Changed|
|Callout.Top auto edge.chromium.png|2307| Changed|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.Bottom right edge.chromium.png|1596| Changed|
|Callout.Left bottom edge.chromium.png|3022| Changed|

</details>


  <details><summary><b>Keytip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Keytip.Offset.chromium.png|121| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Events.chromium.png|2| Changed|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUIV0 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: package.json (hunk 1 above)
Comment (fabricteam) [inline package.json:365]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33145&commitId=43060b179863d61e4857731f22dfe4e58620068c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|8| Changed|
|Avatar Converged.badgeMask.chromium.png|2| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|6204| Changed|
|Drawer.overlay drawer full.chromium.png|1147| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Review by JustSlone — APPROVED** [review]

**Review by sopranopillow — APPROVED** [review]
Looks like the charting demo is broken, unless it was already like that? If so, approving for the v8 website

**Comment by AtishayMsft** [comment]
> Looks like the charting demo is broken, unless it was already like that? If so, approving for the v8 website

Correct. Both charting and experiments demo site are not showing up for the deployed PR. Looks like PR is not able to pick up the changes due to restructuring of the package.
We are tracking this and will follow up if it continues to be broken in master.

File: package.json (hunk 1 above)
Comment (AtishayMsft) [reply package.json:365]: This is unrelated to the PR changes.

Comment (AtishayMsft) [reply package.json:365]: This is unrelated to the PR changes. Diffs reported in line chart is noise.

**Review by Hotell — APPROVED** [review]

**Review by krkshitij — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -147,7 +147,7 @@ packages/react-components/react-aria @microsoft/teams-prg
 packages/react-components/react-aria/library @microsoft/teams-prg
 packages/react-components/react-aria/stories @microsoft/teams-prg
 packages/react-cards @microsoft/cxe-red @khmakoto
-packages/react-charting @microsoft/charting-team
+packages/charts @microsoft/charting-team
```
[the remark this answers is not in the document]
Comment (AtishayMsft) [reply .github/CODEOWNERS]: Updated. Thanks

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Pull request merged**